utils/openwrt-exporter/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2013-2017 OpenWrt.org
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwrt-exporter
```

Comment:
Ok I wasn't to sure about that, the work on most of the lua code base hasn't been done by me, I just added wifi and packaging. Thanks to @jschornick and @rbo!

I went with: `prometheus-node-exporter-lua` for now. No hard feelings about to change that later. Debian uses `prometheus-node-exporter` for the golang clone of that lua script